.vscode/settings.json

```diff
@@ -16,7 +16,7 @@
 		}
 	},
 	"[typescript]": {
-		"editor.defaultFormatter": "biomejs.biome",
+		"editor.defaultFormatter": "esbenp.prettier-vscode",
```

Comment:
What's the reason for this change? 🤔 Biome should also format typescript files in this project